packages/components/src/ui/ariakit-tooltip/index.tsx

```diff
@@ -53,21 +56,20 @@ function AriaToolTip( props: ToolTipProps ) {
 		} );
 	}
 
-	const tooltipState = useTooltipState( {
+	const tooltipStore = useTooltipStore( {
+		// @ts-ignore
 		placement: computedPlacement,
 		timeout: delay,
 	} );
 
 	return (
 		<>
-			<TooltipAnchor as={ Slot } described state={ tooltipState }>
-				{ children }
-			</TooltipAnchor>
+			<TooltipAnchor render={ children } store={ tooltipStore } />
```

Comment:
We were able to remove `Slot` from `radix-ui` in favour of `render` thanks to info shared here: https://github.com/WordPress/gutenberg/pull/48440#discussion_r1217721905

There is a temporary `ts-ignore` added due to errors related to `placement` (only occurring after the slot to render change). I'll need to investigate this further